### PR TITLE
Update categories.txt

### PR DESCRIPTION
**Pull request by ibraheem5000**
Update Arabic Local & fix multiple mistakes.

### CODE REVIEW

**Comment by mpimenov** [comment]
Hello. You need to fill out the CLA before we can merge this.
http://maps.me/cla/

**Comment by ibraheem5000** [comment]
Done

**Comment by ibraheem5000** [comment]
Done

**Comment by ibraheem5000** [comment]
@mpimenov 
I filled out the CLA ...
Any thing else?

**Comment by mpimenov** [comment]
@ibraheem5000 No, that's all. I'll review it within an hour from now.

File: data/categories.txt [hunk 1]
```diff
@@ -68,7 +68,7 @@ sk:Jedlo
 @transport
 en:Transport
 ru:Транспорт
-ar:النقل
+ar:مواصلات|نقل
```
Comment (mpimenov) [inline data/categories.txt]: Please add the '^' sing in front of the preferable translation.

File: data/categories.txt [hunk 2]
```diff
@@ -100,7 +100,7 @@ sk:Doprava
 @fuel
 en:2Gas
 ru:3^Заправка
-ar:1الغاز
+ar:1وقود
```
Comment (mpimenov) [inline data/categories.txt]: Is it true that typing one letter is enough to start suggesting "Fuel" here? If not, please remove the '1'.

File: data/categories.txt [hunk 3]
```diff
@@ -261,7 +261,7 @@ sk:Pamätihodnosť
 @entertainment
 en:Entertainment
 ru:Развлечения
-ar:التسلية
+ar:ترفيه وتسلية
```
Comment (mpimenov) [inline data/categories.txt]: If these are two separate synonyms, please add the '|' sign between them.

File: data/categories.txt [hunk 4]
```diff
@@ -1183,7 +1183,7 @@ sw:^Uzuri saluni
 shop-greengrocer|@shop
 en:4^Greengrocer's|greengrocers|3grocery|U+1F345|U+1F346|U+1F33D|U+1F360|U+1F348|U+1F347|U+1F349|U+1F34A|U+1F34C|U+1F34D|U+1F34E|U+1F34F|U+1F350|U+1F351|U+1F353
 ru:^Овощи и фрукты|овощи|фрукты|2магазин
-ar:^محل خضراوات|متجر
+ar:^متجر خضروات وفواكه|خضار|فاكهة|متجر
```
Comment (mpimenov) [inline data/categories.txt]: Here and in several other places:
Probably the word "متجر" is redundant in the last synonym: remember, a user needs to type the whole phrase before we recognize it as a synonym.

For example, the Russian synonyms for this item one line above read "vegetables and fruit|vegatables|fruit|2shop".

File: data/categories.txt (hunk 2 above)
Comment (ibraheem5000) [inline data/categories.txt]: Yes, One letter is enough

File: data/categories.txt (hunk 3 above)
Comment (ibraheem5000) [inline data/categories.txt]: No, That's ONE synonyms

File: data/categories.txt (hunk 1 above)
Comment (ibraheem5000) [inline data/categories.txt]: Ok, 
I will add the '^' sign front of the preferable translation.
Thanks
But why sometimes I see this sign (^) in translations that have one synonym ???

File: data/categories.txt (hunk 3 above)
Comment (mpimenov) [inline data/categories.txt]: Ok.

File: data/categories.txt (hunk 1 above)
Comment (mpimenov) [inline data/categories.txt]: They are not needed but they won't hurt if we add more translations later.
The main reason for adding them was to make sure that group translations (starting with '@') don't break whatever includes them.

File: data/categories.txt (hunk 4 above)
Comment (ibraheem5000) [inline data/categories.txt]: Ok 
I understand

Thanks

**Comment by ibraheem5000** [comment]
@mpimenov 
Can you review last update please?

Thanks

**Comment by ibraheem5000** [comment]
@mpimenov 

!!!!!!!!

File: data/categories.txt [hunk 5]
```diff
@@ -608,7 +608,7 @@ sw:^Hoteli|restorenti|mgahawa|chakula
 amenity-fuel|@fuel
 en:^Gas Station|3fuel|petrol|petrol station|U+26FD
 ru:Бензин|топливо|азс
-ar:^محطة وقود|وقود|غاز
+ar:^محطة وقود|وقود|غاز|سولار|ديزل|بنزين|بترول
```
Comment (mpimenov) [inline data/categories.txt]: One last comment: note the amenity-fuel|@fuel at Line 608.
All synonyms from @fuel will be appended to amenity-fuel, so there is no need in duplicating them.

Comment (ibraheem5000) [inline data/categories.txt]: Aha !!!
Ok
I removed duplicated synonyms!
Thanks

**Pull request merged**